Letterbook.IntegrationTests/PostsTests.cs

```diff
@@ -129,48 +130,47 @@ public async Task CanDraftNote(Func<Profile, PostDto> generate)
 
 		if(dto.Audience.Count > 0 || dto.AddressedTo.Count > 0)
 		{
-			var cancel = new CancellationTokenSource(TimeSpan.FromMilliseconds(1000));
-			var stop = false;
-
 			const string expected = "urn:message:Letterbook.Workers.Contracts:ActivityMessage send";
-			var spans = await _host.Spans.Where(a => a.TraceId == traceId).TakeWhile(s =>
-			{
-				if (stop) return false;
-				if (s.DisplayName == expected)
-					stop = true;
-				return true;
-			}).ToListAsync(cancel.Token);
-			Assert.Contains(expected, spans.Select(s => s.DisplayName));
+			await Traces.AssertSpan(traceId, _host.Spans, expected);
 		}
 
-		try
-		{
-			// Wait a little while for any subsequent work to happen, then check for exceptions
-			var otherSpans = new CancellationTokenSource(TimeSpan.FromMilliseconds(350));
-			var span = await _host.Spans
-				.Where(a => a.TraceId == traceId)
-				.FirstOrDefaultAsync(span => span.Status == ActivityStatusCode.Error, otherSpans.Token);
-			Traces.SpanIsNotError(span);
-		}
-		// We expect the otherSpans timeout to trigger. This would mean no error spans were found, which is the desired result.
-		catch (TaskCanceledException)
-		{}
-		catch (OperationCanceledException)
-		{}
+		await Traces.AssertNoTraceErrors(traceId, _host.Spans);
+	}
+
+	[Fact(DisplayName = "Should draft a note without publishing")]
+	public async Task CanDraftNote()
```

Comment:
Note to self: this test and the one below seem to be confused :upside_down_face: 

Actual scenarios to cover:
1. draft without publishing (doesn't send any AP message)
2. publish an existing draft (does send AP message)